protected/models/SurveyLanguageSetting.php

```diff
@@ -92,6 +92,8 @@ public function relations()
     public function rules()
     {
         return [
+            ['surveyls_survey_id','required', 'on' => 'insert'],
```

Comment:
Yes, we always need it .... else we can remove it .... some borken system : must be fixed.